near-contract-standards/src/fungible_token/core_impl.rs

```diff
@@ -33,6 +33,25 @@ pub trait FungibleTokenReceiver {
     ) -> PromiseOrValue<U128>;
 }
 
+#[ext_contract(ext_fungible_token)]
+pub trait FungibleTokenContract {
+    fn ft_transfer(&mut self, receiver_id: AccountId, amount: U128, memo: Option<String>);
```

Comment:
Should we use `ValidAccountId` for consistency? I wish we didn't need to do conversion from `AccountId` to `ValidAccountId`, so you don't have to every operate on just `AccountId` unless you do sting operations.
```suggestion
    fn ft_transfer(&mut self, receiver_id: ValidAccountId, amount: U128, memo: Option<String>);
```

Reply:
yeah, in smart contract, we usually are after validation, so converting back to `ValidAccountId` is usually just an unnecessary step. If you think it's better to have it with a type system, then feel free to commit the suggestion. I don't have a strong opinion.

Reply:
Yeah, let's use `AccountId` then in 2 places below